demo/create_voice.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+import asyncio
+import os
+from dotenv import load_dotenv
+from lmnt.api import Speech
+
+load_dotenv() # Don't forget to add your LMNT API key to .env.
+
+async def main(args):
+    async with Speech(os.getenv('LMNT_API_KEY')) as s:
```

Comment:
nit: suggest using the full word `speech` for the variable name here and rest of PR, just because they're demos and it'll be clearer to a new dev that this is the speech instance they're working with.